t/001Basic.t

```diff
@@ -38,3 +38,7 @@ $throttler->try_push() for (1..3);
 is($throttler->try_push(), 0, "rejected before sleep");
 sleep(2);
 is($throttler->try_push(), 1, "1st item after sleep");
+
+$throttler->try_push() for (1..3);
+is($throttler->try_push( 'force' => 0 ), 0, 'no force = reject/no increment');
+is($throttler->try_push( 'force' => 1 ), 0, 'force = reject/increment');
```

Comment:
Would be interesting to see that "force" actually does something useful here, the two tests actually don't detect any difference (a non-existent option like ( "gobble" => 1 ) would yield the same result), is there a way we could check that the value of the last bucket has in fact increased?